docs/getting-started.md

```diff
@@ -63,6 +63,7 @@ curl localhost:10101/index/repository -X POST
 ``` response
 {"success":true}
 ```
+The index name must be 64 characters or less, start with a letter, and consist only of lowercase alphanumeric characters or _-.
```

Comment:
I think it might still need the back ticks around the underscore Dash

Reply:
Whoops my bad